lib/MusicBrainz/Server/Entity/Collection.pm

```diff
@@ -44,6 +45,7 @@ around TO_JSON => sub {
     $json->{editor} = $self->editor ? $self->editor->TO_JSON : undef;
     $json->{public} = boolean_to_json($self->public);
     $json->{description} = $self->description;
+    $json->{descriptionHtml} = format_wikitext($self->description);
```

Comment:
Would mildly prefer `description_html` just for consistency with Moose attributes being snake case.